hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/receiver/kafka/KafkaSingleThreadedMessageReceiver.java

```diff
@@ -152,6 +152,10 @@ public void stop() {
             consumer.close();
         } catch (IllegalStateException ex) {
             // means it was already closed
+        } catch (InterruptException ex) {
+            logger.warn("InterruptException occurred during closing consumer: " + ex);
+        } catch (KafkaException ex) {
+            logger.warn("KafkaException occurred during closing consumer: " + ex);
```

Comment:
We probably want to log a stack trace. Therefore, a better option is to do it this way:
```java
logger.warn("KafkaException occurred during closing consumer.", ex);
```